pasqal_cloud/__init__.py

```diff
@@ -390,7 +404,6 @@ def add_jobs(self, batch_id: str, jobs: List[CreateJob]) -> Batch:
 
         Raises:
             JobCreationError, which spawns from a HTTPError.
-            BatchAlreadyCompleteError.
```

Comment:
should raise OnlyCompleteOrOpenCanBeSet instead, could you add it ?

Reply:
I think you're mistaken. This is the add jobs method.

Reply:
Indeed, mb